test/ServerComparison.FunctionalTests/HelloWorldTest.cs

```diff
@@ -32,7 +32,9 @@ public static TestMatrix TestVariants
         public async Task HelloWorld(TestVariant variant)
         {
             var testName = $"HelloWorld_{variant.Server}_{variant.Tfm}_{variant.Architecture}_{variant.ApplicationType}";
-            using (StartLog(out var loggerFactory, testName))
+            using (StartLog(out var loggerFactory,
```

Comment:
Any reason we don't want to set the level to trace in all cases? It's a lot more verbose but that's what we have test logs for.

Also, these tests look like they are using outdated infrastructure for functional test logging, care to update them?

Reply:
Yes, IIS logs its whole config file... Debug might be a better default in general.

I need StartLog for now to set the log level but can look at removing it when we revert this.